hyperstart/mock/hyperstart.go

```diff
@@ -257,6 +262,43 @@ func (h *Hyperstart) handleCtl() {
 
 		h.SendMessage(hyper.INIT_ACK, nil)
 
+		sendPAE := false
```

Comment:
Instead of this, I would add a method simlar to SendExitStatus() that sends an event through the CTL channel so the client code can decide when it wants the process to end instead of an automatic end as soon as it has been created. That will allow testing multiple processes running at the same time.